base/multi.jl

```diff
@@ -368,7 +368,19 @@ const LPROC = LocalProcess()
 
 const HDR_VERSION_LEN=16
 const HDR_COOKIE_LEN=16
+
+"""
+    Base.cluster_cookie() -> cookie
+
+Returns the cluster cookie, by default the `LocalProcess` cookie.
```

Comment:
Reference to `LocalProcess` can be removed. It is not really mentioned in the docs anywhere and should also be probably unexported - not really relevant/useful in user code.